recipes/vulkan-utility-libraries/all/conanfile.py

```diff
@@ -0,0 +1,174 @@
+import os
+
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.apple import fix_apple_shared_install_name, is_apple_os
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.env import VirtualBuildEnv
+from conan.tools.files import copy, get, rm, rmdir
+from conan.tools.microsoft import is_msvc
+from conan.tools.scm import Version
+
+required_conan_version = ">=1.55.0"
+
+
+class VulkanUtilityLibrariesConan(ConanFile):
+    name = "vulkan-utility-libraries"
+    description = "Utility libraries for Vulkan developers"
+    license = "Apache-2.0"
+    topics = ("vulkan",)
+    homepage = "https://github.com/KhronosGroup/Vulkan-ValidationLayers"
+    url = "https://github.com/conan-io/conan-center-index"
+
+    package_type = "static-library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "fPIC": True,
+    }
+    short_paths = True
```

Comment:
```suggestion
    short_paths = True
```

What error causes without `short_paths` ?

Reply:
It's probably just a leftover from the `vulkan-validationlayers` recipe I based this one on. Removed it.